package.json

```diff
@@ -29,9 +29,17 @@
     "url": "https://github.com/utopias/hello-world/issues"
   },
   "dependencies": {
-    "body-parser": "^1.0.0",
```

Comment:
Lets go back to the root version here, so devs can see we arent pinning to a specific version.